index.js

```diff
@@ -88,6 +89,11 @@ Object.defineProperties(MediaSession.prototype, {
             }
             return [];
         }
+    },
+    channels: {
+        get: function () {
+            return this.pc._remoteDataChannels.concat(this.pc._localDataChannels);
```

Comment:
don't use those please... they're just a workaround for both browsers garbage collecting data channels (unfixed for 1+ year now)